server/src/domain/handler.rs

```diff
@@ -101,7 +101,7 @@ impl From<bool> for GroupRequestFilter {
 pub struct CreateUserRequest {
     // Same fields as User, but no creation_date, and with password.
     pub user_id: UserId,
-    pub email: Email,
+    pub email: Option<Email>, // Required, but could be passed in attributes
```

Comment:
Hard disagree. At this layer, the email should have been taken out of the attributes and into the field. We should have a single, canonical representation in the DB, and it's with email as a column of user, not in the attributes.

I think the correct place to do the translation is in the GrahpQL layer, in mutation.rs

Reply:
Same for `display_name`, pick one, either a column or an attribute, but not both.

Reply:
I'll have a look if I have time to change the server side to handle that for you, outside of this PR

Reply:
Found time to do it. You can now send the email and display name as attributes _or_ main field.